docs/INTEGRATION.md

```diff
@@ -9,24 +13,24 @@
 To run headless Cypress tests, enter the following command from the root of the project:
 
 ```shell
-yarn test-cypress
+yarn test-Cypress
 ```
 
 This will automatically start the React and proxy servers and run the Cypress tests, in which results are logged to the console. After running the tests, the servers and process will close.
 
 ## Edit local configuration
 
-By default, cypress will run tests using the configuration defined in [cypress.json](/cypress/cypress.json).
+By default, Cypress will run tests using the configuration defined in [Cypress.json](/Cypress/Cypress.json).
```

Comment:
this does not exist in our code base, does it?

`find . -iname "cypress.json"` --> nothing

Reply:
Good find - this should be `cypress.config.ts`